gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/service/JavaServiceDistributionPlugin.java

```diff
@@ -99,9 +102,15 @@ public void apply(Project project) {
 
         // Create configuration to load executable dependencies
         Configuration launcherConfig = project.getConfigurations().create("goJavaLauncherBinary");
-        project.getDependencies().add(launcherConfig.getName(), GO_JAVA_LAUNCHER);
+        project.getDependencies().add(launcherConfig.getName(), getGoJavaLauncherCoordinate(project, GO_JAVA_LAUNCHER));
         Configuration initConfig = project.getConfigurations().create("goInitBinary");
-        project.getDependencies().add(initConfig.getName(), GO_INIT);
+        project.getDependencies().add(initConfig.getName(), getGoJavaLauncherCoordinate(project, GO_INIT));
+
+        project.getRepositories().mavenCentral(m -> {
```

Comment:
any reason we need this instead of relying on whatever repository the project has configured? I can imagine this could be problematic for internal usage since we don't want to be loading directly from maven central

Reply:
removed